blocks/serialization.py

```diff
@@ -62,9 +62,11 @@ def __call__(self, obj):
         if isinstance(obj, SharedVariable):
             super(PersistentParameterID, self).__call__(obj)
             if hasattr(obj.tag, 'annotations'):
-                name = BRICK_DELIMITER.join(
-                    [brick.name for brick
-                     in get_brick(obj).get_unique_path()] + [obj.name])
+                name = '{}.{}'.format(
```

Comment:
Now that you mention it, it already exists actually: https://github.com/mila-udem/blocks/blob/master/blocks/select.py#L39, so we should probably re-use that one.